client/webserver/site/src/html/settings.tmpl

```diff
@@ -17,12 +17,30 @@
         Show pop-up notifications
       </label>
     </div>
-    <div{{if not .UserInfo.Authed}} class="d-hide"{{end}}>
-      <p>
+    <div>
+      <div {{if eq (len .UserInfo.Exchanges) 0}} class="d-hide"{{end}}>
+        <div>
+          {{range $host, $xc := .UserInfo.Exchanges}}
+            <div>
+              Dex Address: {{$host}}
+              {{if $xc.IsAuthed}}
```

Comment:
I'm not sure we want to add IsAuthed as a field to the Exchanges struct, at least not for this display logic.  Why not display AcctID regardless?